packages/roosterjs-editor-dom/lib/selection/setHtmlWithSelectionPath.ts

```diff
@@ -0,0 +1,29 @@
+import createRange from './createRange';
+import { NodeType, SelectionPath } from 'roosterjs-editor-types';
+
+/**
+ * Restore inner Html of a root element from given html string. If the string contains selection path,
+ * remove the selection path and return a range represented by the path
+ * @param root The root element
+ * @param html The html to restore
+ * @returns A selection range if the html contains a valid selection path, otherwise null
+ */
+export default function getHtmlWithSelectionPath(rootNode: HTMLElement, html: string): Range {
```

Comment:
typo: setHtmlWithSelectionPath

Reply:
good catch

Reply:
actually, if the function returns a range, the name is a bit misleading -- maybe something like `restoreHtmlAndGetSelectionPath`?